packages/zoe/src/zoe.js

```diff
@@ -718,7 +718,10 @@ const makeZoe = (additionalEndowments = {}) => {
                   success,
                   details`invites must be issued by the inviteIssuer.`,
                 );
-                return invite;
+                return {
+                  invite,
+                  instanceRecord: { ...instanceRecord, publicAPI },
```

Comment:
The instanceRecord should already include the publicAPI, right?

Reply:
yep.

Reply:
The one in the table might by this point, but this is still the local record we created in order to initialize the instanceRecord. It has an explicit `publicAPI: undefined` from line 704.

Reply:
Doesn't this line happen right before this?

```instanceTable.update(instanceHandle, { publicAPI });```

The updates are synchronous, so the publicAPI is in the instanceRecord.

With the `initPublicAPI` change now merged, this has all been changed, but either way, explicitly adding the publicAPI is unnecessary.

Reply:
Oh, I see what you're saying, shouldn't we just get the instanceRecord again to return it?

Reply:
I'll merge with the initPublicAPI change and see what shakes out.

But just to check my understanding, are you saying that calling
```js
instanceTable.update(instanceHandle, { publicAPI });
```
will side effect the local `instanceRecord` object? I certainly agree that the instanceRecord in the table has been changed at this point, but I didn't think that would impact the local copy of the object. 

table's implementation of `update()` uses the spread operator to copy fields out of its `partialRecord` parameter to build a new record to store in the table.

Reply:
It won't affect the object assigned to the variable `instanceRecord` (no side effects ftw!), but the instanceRecord in the table is updated synchronously, so we could just get that again. I didn't realize you were only talking about the object assigned to `instanceRecord`.